app/bg/background.js

```diff
@@ -127,6 +127,37 @@ let onCommandListener = function(cmd) {
 } //onCommandListener()
 chrome.commands.onCommand.addListener(onCommandListener);
 
+// When a window is created
+chrome.windows.onCreated.addListener(function(window){
+  // Leave TabFern window alone...
+  if (window.id === me.viewWindowID) return;
```

Comment:
I am not sure if viewWindowID will always exist when this function is called.  E.g., I don't know if Chrome will give us window-creation messages for the first window that opens before the TF popup opens.  Would you please add a check that viewWindowID is defined before using it?